src/components/TeamMemberList.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import TeamMember from "./TeamMember";
+import axios from "axios";
+
+const TeamMemberList = (props) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
+      setUsers(response.data);
+    };
+    fetchUsers();
+  },[]);
+
+  return (
+    <>
+      {users.map((user) => {
+        return (
+          <TeamMember
+            key={user.id}
+            firstName={user.firstName}
```

Comment:
you are passing 4 props with user data, would it be easier to pass just `user`? :)

Reply:
Changed to user as suggested and changed TeamMember component to support it. I left key={user.id} because I read that this should be added by higher component - so in this case TeamMemerList.